tests/core/full_node/test_mempool.py

```diff
@@ -282,7 +253,7 @@ async def test_basic_mempool_manager(self, bt, two_nodes_mempool, wallet_a, self
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 0, MempoolInclusionStatus.PENDING),
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 1, MempoolInclusionStatus.PENDING),
             # the absolute height and seconds tests require fresh full nodes to
-            # run the test on. The fixture (two_nodes_mempool) creates 3 blocks,
+            # run the test on. The fixture (one_node_one_block) creates 3 blocks,
```

Comment:
the name of this fixture suggests that it only creates one block (compared to 3 block by `two_nodes_mempool`). If that's right, I would expect the heights below need updating as well.

Reply:
It only creates one block, i'll update the comment